thirdeye/thirdeye-frontend/app/pods/components/modals/create-event-modal/component.js

```diff
@@ -0,0 +1,50 @@
+/**
+ * Component displaying the event creation inside a modal
+ * @module components/modals/create-event-modal
+ * @property {Function} onSave          - save the events
+ * @property {Function} onExit          - toggle off the modal
+ * @example
+ {{modals/create-event-modal
+   showCreateEventModal=showCreateEventModal
+ }}
+ * @exports create-event-modal
+ * @author jihzhang
+ */
+
+import Component from '@ember/component';
+import {
+  get, set, getProperties
+} from '@ember/object';
+import {inject as service} from '@ember/service';
+
+import fetch from 'fetch';
+
+import createEventApi from 'thirdeye-frontend/utils/api/create-event';
+
+export default Component.extend({
+
```

Comment:
You might be missing this class to style the modal correctly. See `app/pods/components/modals/entity-mapping-modal/component.js`

```
  /**
   * Custom classes to be applied to the entity modal table
   */
  classes: {
    theadCell: "te-modal__table-header"
  },

```

Reply:
fixed